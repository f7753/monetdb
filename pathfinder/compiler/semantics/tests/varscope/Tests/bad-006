#!/bin/bash

pf -s4 <<'EOT'
for $a in ($b,2) for $b in (3,4) return ($a, $b)
EOT
echo $?
