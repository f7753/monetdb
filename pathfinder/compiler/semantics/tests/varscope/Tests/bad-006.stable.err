stderr of test 'bad-006` in directory 'compiler/semantics/tests/varscope` itself:


# 12:43:25 >  
# 12:43:25 >  Mtimeout -timeout 60 ./bad-006 bad-006 
# 12:43:25 >  

variable(s) out of scope or unknown: at (1,13-1,13): $b
variable(s) out of scope or unknown: erroneous variable references reported above


# 12:43:25 >  
# 12:43:25 >  Done.
# 12:43:25 >  

