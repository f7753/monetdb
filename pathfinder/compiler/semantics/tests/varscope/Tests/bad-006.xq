for $a in ($b,2) for $b in (3,4) return ($a, $b)
