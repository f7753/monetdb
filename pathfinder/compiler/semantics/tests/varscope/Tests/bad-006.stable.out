stdout of test 'bad-006` in directory 'compiler/semantics/tests/varscope` itself:


# 12:43:25 >  
# 12:43:25 >  Mtimeout -timeout 60 ./bad-006 bad-006 
# 12:43:25 >  

8

# 12:43:25 >  
# 12:43:25 >  Done.
# 12:43:25 >  

